docs/edsl.html

```diff
@@ -285,19 +285,20 @@ <h3 id="max-pool-1d">Max Pool 1D</h3>
 \Large
 \textcolor{red}{O[n]}
 \textcolor{yellow}{=}
-\textcolor{green}{\max}\textcolor{magenta}{_{0 \leq j &lt; 2}}
-\textcolor{cyan}{I[2i + j]}</p>
+\textcolor{green}{\max}
+\textcolor{magenta}{_{0 \leq j \lt 2}}
+\textcolor{cyan}{I[2i + j]}
 \]
 </p>
 
 <p>
 \[
 \Large
-  if (\textcolor{magenta}{j &lt; 2}) {{
+  \mathrm{add\_constraints} (\textcolor{magenta}{j \lt 2}) {\mathrm{\{}
```

Comment:
I meant `add_constraints` as in the code block above, so the 2 lines
```
  O(i) >= I(2 * i + j);
  O.add_constraints({j<2});
```
Probably the entire second line should be magenta. I'm not quite sure how mathjax handles multiline equations; if it were LaTeX I'd be using AMS' `\begin{align*}` environment instead of `\[`, but I doubt that works in mathjax. You can try `\\` and `\newline` perhaps? Worst case scenario make 2 equations back-to-back.